template/docs/_static/css/custom.css

```diff
@@ -13,9 +13,28 @@ html[data-theme="dark"] {
  */
 .bd-header .navbar-nav li a.nav-link .icon-link-image {
     filter: var(--scipp-filter-header-icon);
+    height: 1em;
 }
 
 .bd-header .navbar-nav li a.nav-link .icon-link-image:hover {
     /* match primary color */
-    filter: hue-rotate(85deg) saturate(0.829) brightness(0.945);
+    filter: hue-rotate(153deg) saturate(1.28) brightness(0.7);
+}
+
+.search-button-field {
+    border: none;
+    border-radius: 0;
+    background: inherit;
+    color: inherit;
+}
+
+.search-button-field:hover {
+    border: none;
+    color: var(--pst-color-link-hover);
+    box-shadow: 0 max(3px,.1875rem,.12em) 0 var(--pst-color-link-hover);
+}
+
+.search-button-field > span {
+    display: none;
+    visibility: hidden;
```

Comment:
Will we get into a constant uphill battle by changing the CSS too much, having to fix frequently when the theme updates?

Reply:
That is a risk. I would hope that they don't constantly change the layout or css classes. But I can't tell you how often we will need to apply fixes.